Source/ACE/Entity/DebugObject.cs

```diff
@@ -77,6 +77,7 @@ public DebugObject(ObjectGuid guid,  BaseAceObject baseAceObject)
                 // Creating from a pcap of the weenie - this will be set by the loot generation factory. Og II
                 this.PhysicsData.PhysicsDescriptionFlag &= ~PhysicsDescriptionFlag.Parent;
                 this.GameData.ValidLocations = (EquipMask)baseAceObject.ValidLocations;
+                this.GameData.IconOverlay = (ushort)baseAceObject.IconOverlayId;
```

Comment:
GameData.IconOverlay is defined as a ushort.   The iconOverlayId is defined in BaseAceObject as an int.   It is an int in the data as well. so do I take out the 0x06000000 from the number in the database and just store last 4 up to FFFF which will fit in a ushort or change or change gameData?   I am assuming strip the number - I need to make sure the writer is adding the 0x6000000 back to to iconoverlay and underlay first.   Let me know and I will fix it.